src/server/worker.cc

```diff
@@ -322,6 +327,10 @@ Status Worker::AddConnection(redis::Connection *c) {
   return Status::OK();
 }
 
+std::map<int, redis::Connection *> Worker::GetConnection() {
+  return std::move(conns_);
```

Comment:
Is it intentional to clear `conns_` after this function? IMHO it is weird to move class field in a getter function.